arcgis-ios-sdk-samples/Utility Network/Configure subnetwork trace/ConfigureSubnetworkTraceViewController.swift

```diff
@@ -0,0 +1,404 @@
+        case .conditions:
+            return traceConditionalExpressions.count
+        }
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        switch Section.allCases[indexPath.section] {
+        case .switches:
+            let cell = tableView.dequeueReusableCell(withIdentifier: "SwitchCell", for: indexPath)
+            switch Switches.allCases[indexPath.row] {
+            case .barriers:
+                cell.accessoryView = barriersSwitch
+                cell.textLabel?.text = Switches.barriers.label
+            case .containers:
+                cell.accessoryView = containersSwitch
+                cell.textLabel?.text = Switches.containers.label
+            }
+            return cell
+        case .conditions:
+            let cell = tableView.dequeueReusableCell(withIdentifier: "ConditionCell", for: indexPath)
+            cell.textLabel?.text = expressionToString(expression: traceConditionalExpressions[indexPath.row])
+            return cell
+        }
+    }
+    
+    func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
+        if indexPath.section == Section.conditions.rawValue && indexPath.row != 0 {
+            // Enable editing for the rest of conditions section.
+            return true
+        } else {
+            // Disable editing for the first row of conditions section and other rows.
+            return false
+        }
+    }
+    
+    func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
+        if indexPath.section == Section.conditions.rawValue && editingStyle == .delete {
+            traceConditionalExpressions.remove(at: indexPath.row)
+            tableView.deleteRows(at: [indexPath], with: .fade)
+            // Reload footer label.
+            chainedConditionsLabel.text = expressionString
+        }
+    }
+}
+
+/// An extension of `AGSUtilityCategoryComparisonOperator` that returns a human readable description.
+/// - Note: You may also create a `AGSUtilityCategoryComparison` with
+///         `AGSUtilityNetworkDefinition.categories` and `AGSUtilityCategoryComparisonOperator`.
+private extension AGSUtilityCategoryComparisonOperator {
+    var title: String {
+        switch self {
+        case .exists: return "Exists"
+        case .doesNotExist: return "DoesNotExist"
+        @unknown default: return "Unknown"
+        }
+    }
+}
+
+/// An extension of `AGSUtilityAttributeComparisonOperator` that returns a human readable description.
+extension AGSUtilityAttributeComparisonOperator: CaseIterable {
+    // Manually set the maximum raw value of cases to facilitate protocol conformation.
+    private static let maxCaseValue = Self.doesNotIncludeAny.rawValue
+    
+    public static let allCases = {
+        return (0...maxCaseValue).map { AGSUtilityAttributeComparisonOperator(rawValue: $0)! }
+    }()
```

Comment:
To solve the comment https://github.com/Esri/arcgis-runtime-samples-ios/pull/897#discussion_r480242508, I decided to make this change, but still don't feel good about it. While it works, it has a few drawbacks

- It cannot detect if the `AGSUtilityAttributeComparisonOperator ` enum changes in the future, due to manually assigned
- By not being private, it would affect/pollute future usage of the enum

Wait for Phil's thoughts.